dbfit-java/core/src/main/java/dbfit/api/DbStoredProcedureCall.java

```diff
@@ -52,14 +51,8 @@ void bindParametersTo(StatementExecution cs) throws SQLException {
     }
 
     public StatementExecution toStatementExecution() throws SQLException {
-        String sql = toSqlString();
-        PreparedStatement ps = environment.getConnection().prepareCall(sql);
-        StatementExecution cs;
```

Comment:
> May be we should rename cs now. Presumably cs was once a CallableStatement.

It's coming from `createCallExecution` and is indeed backed by `CallableStatement` - so callable statement is so bad association... Yet maybe just `statement` would be a bit more clear (short abbreviations are sometimes obscure).